src/client/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -104,6 +104,19 @@ public PublicClientApplicationBuilder WithDefaultRedirectUri()
             return this;
         }
 
+        /// <summary>
+        /// Configures the public client application to use multi cloud support.
+        /// It enables applications to pass in a default public cloud authority to the library and can still get tokens for resources from national clouds.
+        /// </summary>
+        /// <returns>A <see cref="PublicClientApplicationBuilder"/> from which to set more
+        /// parameters, and to create a public client application instance</returns>
+        public PublicClientApplicationBuilder WithMultiCloudSupport()

```

Comment:
For consistency, please have this as 

`WithMultiCloudSupport(bool enabled)`

Reply:
Done